src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2856,3 +2856,9 @@ Include\ or\ exclude\ cross-referenced\ entries=Include or exclude cross-referen
 Would\ you\ like\ to\ include\ cross-reference\ entries\ in\ the\ current\ operation?=Would you like to include cross-reference entries in the current operation?
 Entries\ copied\ successfully,\ including\ cross-references.=Entries copied successfully, including cross-references.
 Entries\ copied\ successfully,\ without\ cross-references.=Entries copied successfully, without cross-references.
+
+All\ patterns=All patterns
+Author\ related=Author related
+Editor\ related=Editor related
+Title\ related=Title related
+BibEntry\ fields=BibEntry fields
```

Comment:
I totally overlooked the last entry while reviewing. (Nearly) all other translations use "entry" only.

Can you file a full-up PR replacing "BibEntry" by "Entry"/"entry"? (I found two hits in the language file)